cns/restserver/homeazmonitor.go

```diff
@@ -154,7 +154,14 @@ func (h *HomeAzMonitor) Populate(ctx context.Context) {
 		h.update(returnCode, returnMessage, cns.HomeAzResponse{IsSupported: true})
 		return
 	}
-	h.update(types.Success, "Get Home Az succeeded", cns.HomeAzResponse{IsSupported: true, HomeAz: azResponse.HomeAz})
+	// validate APIVersion value
+	if !azResponse.Valid() {
```

Comment:
Is it ok to IsValid check after checking AZ? Should we do Valid check before it?

Reply:
imo, its ok. the order does not matter here. the APIversion is for indicating the ipv6 fix which is independent of the value of home az. so it could be home az is 0, but APIversion is either valid or invalid.